### ISSUE TITLE
Cemu not recognized

### ISSUE THREAD

**Issue by ghost** [Berny23/LD-ToyPad-Emulator#58]

**Comment by Whiter645** [comment]
I use Cemu for playing Lego Dimensions because RPCS3 lags too much, i had to do the Zadig workaround to use the emulator on RPCS3, but i did the driver rollback for the Lego READER, now the problem is that the emulator doesn't recognize Cemu, also, my Cemu version is 1.26.2f

**Comment by Whiter645** [comment]
Yeah i am an idiot and opened the issue 2 times and closed the one with the text

**Comment by RetroZoneYT** [comment]
im also having this problem

**Comment by RetroZoneYT** [comment]
Okay found a fix uninstall the drivers and its software 

also RPCS3 Requires Atleast Around Almost Near 3GB And 50% Of CPU And GPU At Default But i dont recormend it if you dont have a low end pc or a unsupported Vulkan Device

**Comment by oodledoodlex** [comment]
Hi I'm having the same issue as the person above, any advice? :)
